examples/all-clusters-app/p6/src/AppTask.cpp

```diff
@@ -104,6 +105,7 @@ using namespace ::chip::DeviceLayer;
 using namespace ::chip::System;
 
 AppTask AppTask::sAppTask;
+static chip::DeviceLayer::DeviceInfoProviderImpl gExampleDeviceInfoProvider;
```

Comment:
This example impl forces a specific set of fixed labels.  It shouldn't really be used for anything; it's a temporary placeholder until platforms implement actual device info providers...

Reply:
@bzbarsky-apple : Thanks for the review. we understand that this is temporary placeholder. But this is required for SVE test execution and final device info providers will be implemented based on Customer/Marketing requirements.